README.md

```diff
@@ -1,2 +1,34 @@
 # comp110-worksheet-5
 Base repository for COMP110 worksheet 5
+
+(a)
+	Iterates through the list searching for duplicates
+
+(b)
+	Because when i and j are in different positions on the same list it returns true
+
+(c)
+	Because it will still iterate through the same positions
+
+(d)
+	Because it will return false less
+
+(e)
+	The quadratic equation is still true as the method of comparing each entry persists
+
+(f)
+	O(n log n)
+	
+	Article title:	TimeComplexity - Python Wiki
+	Website title:	Wiki.python.org
+	URL:	https://wiki.python.org/moin/TimeComplexity
+
+(g)
+	O(n)
+	Because it is the most complex function in the algorithm
+	
+(h)
+	Algorithm 2 as it doesn't wait for the two pointers to meet before returning as false
```

Comment:
Right answer but I don't understand your reason? (The answer I was looking for was to compare the time complexities and note that `O(n log n) < O(n^2)` for large n)